plugins/BadWords/config.py

```diff
@@ -113,6 +113,9 @@ class ReplacementMethods(registry.OnlySomeStrings):
     filtering.  If it's True, however, it will interact poorly with other
     plugins that do coloring or bolding of text.""")))
 
+conf.registerChannelValue(BadWords, 'selfCensor',
+    registry.Boolean(True, _("""Determines whether the bot will filter itself in
+    this channel.""")))
```

Comment:
Just "Determines whether the bot will filter its own message". Variable help strings do not mention "in this channel" because this is indicated by other means (list prefixes, current values in output of `@help list`, etc.)

Reply:
Done